### PR TITLE
More changes to support core.js

### PR DESCRIPTION
**Pull request by KarlCastle**
Now core.js is included everywhere.

Still need to:
 * migrate Language & Localization (to core.js or their own language.js?)
 * migrate BrowserHelper
 * migrate ExtensionLayer
 * pare down LocalStorage/LocalData

### CODE REVIEW

File: js/options/options.js [hunk 1]
```diff
@@ -212,7 +212,8 @@ let Options = (function(){
         Region.populateRegionalSelects();
         
         if (!changelogLoaded) {
-            RequestData.getHttp('changelog.txt', {type: "text/plain"}).then(data => {
+            fetch(chrome.runtime.getURL('changelog.txt')).then(r => r.text())
```
Comment (tfedor) [inline js/options/options.js]: Please refactor back to using RequestData

File: js/options/options.js [hunk 2]
```diff
@@ -335,114 +334,97 @@ let Options = (function(){
     }
 
     self.init = async function() {
-        let settings = SyncedStorage.load();
+        let settings = SyncedStorage.init();
         let currency = fetch(chrome.runtime.getURL('json/currency.json'))
```
Comment (tfedor) [inline js/options/options.js]: Refactor to use layer between fetch

File: js/options/options.js (hunk 1 above)
Comment (KarlCastle) [reply js/options/options.js]: Created a new PR tfedor/AugmentedSteam#150 https://github.com/tfedor/AugmentedSteam/commit/45fb1cf600915364438babc221ac62d57032a1f3

File: js/options/options.js (hunk 2 above)
Comment (KarlCastle) [reply js/options/options.js]: Created a new PR tfedor/AugmentedSteam#150 https://github.com/tfedor/AugmentedSteam/commit/45fb1cf600915364438babc221ac62d57032a1f3

**Pull request merged**